pkg/controller/healthcheck/add.go

```diff
@@ -79,6 +91,12 @@ func RegisterHealthChecks(mgr manager.Manager, opts healthcheck.DefaultAddArgs)
 				HealthCheck:   general.NewSeedDeploymentHealthChecker(openstack.CSISnapshotControllerName),
 				PreCheckFunc:  csiEnabledPreCheckFunc,
 			},
+			// TODO(acumino): Enable this health check in v1.27.
+			{
+				ConditionType: string(gardencorev1beta1.ShootControlPlaneHealthy),
+				HealthCheck:   general.NewSeedDeploymentHealthChecker(openstack.CSISnapshotValidation),
+				PreCheckFunc:  healthCheck,
+			},
```

Comment:
This should be commented out right ?

Reply:
No, the Precheck function is doing that only, it is skipping the health check.

Reply:
Then maybe I don't understand the TODO. The precheck ia already only executed if the `csi-snapshot-validation` is present so what would need to be changed come v1.27. Can you elaborate ?

Reply:
We can't start the health check for this deployment with the release of it, because it will make the shoot status to unhealthy because of the missing snapshot-validation deployment. the status can remain false until the shoot is reconciled, to don't populate our landscapes dashboard with this false positive we skip the health check with the first release and it will be enabled with the upcoming release.